discover/index.html

```diff
@@ -58,41 +58,41 @@ <h3 class="text-left" id="connect"><i class="fab fa-fw fa-mixcloud"></i> Get Con
-  <li><i class="fas fa-fw fa-unlock"></i> <code>Basemap-Topo</code> – Base map of USGS topographic maps
-  <li><i class="fas fa-fw fa-unlock"></i> <code>DOQ 1990s BW</code> – 1-meter statewide black &amp; white aerial photography collected in 1990’s
-  <li><i class="fas fa-fw fa-unlock"></i> <code>HRO 2012 RGB</code> – 12.5-cm natural color aerial photography collected in 2012 of the Wasatch Front
-  <li><i class="fas fa-fw fa-unlock"></i> <code>NAIP {year} RGB</code> – 1-meter statewide natural color aerial photography
+  <li><i class="fas fa-fw fa-lock"></i> <code>Basemap-Hybrid</code> - Base map combination of the overlay base map and Utah imagery service <strong>(licensed)</strong>
+  <li><i class="fas fa-fw fa-unlock"></i> <code>Basemap-Lite</code> – Base map of hillshade, highways and streets, cities, boundaries, etc. (gray, muted tone)
+  <li><i class="fas fa-fw fa-unlock"></i> <code>Basemap-Overlay</code> – Base map of vector features, highways and streets, cities, address points, parcels, etc. (transparent background)
+  <li><i class="fas fa-fw fa-unlock"></i> <code>Basemap-Terrain</code> – Base map of hillshade, highways and streets, parks, forests, water, etc.
+  <li><i class="fas fa-fw fa-unlock"></i> <code>Basemap-Topo</code> – Base map of US Geological Survey topographic maps
+  <li><i class="fas fa-fw fa-unlock"></i> <code>DOQ 1990s BW</code> – 1-meter statewide black-and-white aerial photography, collected in the 1990s
+  <li><i class="fas fa-fw fa-unlock"></i> <code>HRO 2012 RGB</code> – 12.5-cm natural-color aerial photography of the Wasatch Front, collected in 2012 
+  <li><i class="fas fa-fw fa-unlock"></i> <code>NAIP {year} RGB</code> – 1-meter statewide natural-color aerial photography
     <ul class="dotless">
-      <li><em>Available Years: 2006 2009 2011 2014 2016</em>
+      <li><em>Available Years: 2006, 2009, 2011, 2014, 2016</em>
     </ul>
   <li><i class="fas fa-fw fa-unlock"></i> <code>NAIP {year} NRG</code> – 1-meter statewide color infrared aerial photography
     <ul class="dotless">
-      <li><em>Available Years: 2006 2011 2014 2016</em>
+      <li><em>Available Years: 2006, 2011, 2014, 2016</em>
     </ul>
-  <li><i class="fas fa-fw fa-unlock"></i> <code>Hillshade</code> – 10 meter statewide combined hillshade and slope
+  <li><i class="fas fa-fw fa-unlock"></i> <code>Hillshade</code> – 10-meter statewide combined hillshade and slope
 </ul>
 <h3 class="text-left"><i class="far fa-fw fa-thumbs-up"></i> Best Practices</h3>
 <ul>
-  <li>Information about <a href="{{ "/data/googleimagery/" | prepend: site.baseurl }}">transformations between different coordinate systems</a>.
+  <li>Click <a href="{{ "/data/googleimagery/" | prepend: site.baseurl }}">here</a> for more information about transformations between different coordinate systems.
+  <li>Click <a href="{{ "/data/googleimagery/" | prepend: site.baseurl }}">here</a> for more information about using Google Imagery and Discover Services.
```

Comment:
(This comment/question actually applies to the next line, fyi) Should WMTS show the acronym's definition when you hover the mouse over it? (Like it does earlier in the article?)

Also, "for the best performance"--what does this mean? Best performance of what?

Reply:
We try to avoid `click here` links but sometimes it is not avoidable

Reply:
Oh, sorry about that! I was just trying to make the sentences more parallel/consistent! If we prefer it a different way, I can reword or revise back to the original?

Reply:
In this case the inline link doesn't read well, I think click here is ok. Did you mean to duplicate the same link twice?

Reply:
I think performance here is regarding best draw time/perceived performance.

I read the article again and I think those two links can be removed.